lisa_commands/lambda_function.py

```diff
@@ -39,3 +45,29 @@ def lambda_handler(data, _context):
 
     request.submit_slack_request(data, chat_action)
     return request.return_status()
+
+def valid_slack_request(data):
+    """
+    Did this request come from slack
+    data - the data object (dict)
+    """
+    slack_signature = data['headers']['X-Slack-Signature']
+    slack_request_timestamp = data['headers']['X-Slack-Request-Timestamp']
+    request_body = data['body']
+    basestring = f"v0:{slack_request_timestamp}:{request_body}".encode('utf-8')
```

Comment:
Is the use of "basestring" intentional here, or mere convenience? If convenience, another nitpick, but I think it's a special name in python. Even renaming to base_string or base_request_string would evade that (extremely, extremely) minor problem.